Runtime_Support/early_access.md

```diff
@@ -0,0 +1,37 @@
+# Early Access Runtime
+
+**Early Access** is a new feature of App Service that allows for the rapid delivery of updates and new versions of the programing languages/runtime and SDK supported by the platform. As a developer you will get access to the latest new releases, without having to wait for an App Service release cycle to include support.
```

Comment:
**Early Access** is a new feature of Azure App Service that allows for the rapid delivery of updates and new versions of various-language programming runtimes and SDKs supported by the platform. As a developer, you will get access to the latest new releases without having to wait for an App Service release cycle to include support.